components/Editor/index.tsx

```diff
@@ -240,7 +241,7 @@ const Editor = ({ readOnly = false }: Props) => {
       onForkChange(query.fork as string)
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [settingsLoaded && router.isReady])
+  }, [settingsLoaded && router.isReady && isForksLoaded])
```

Comment:
This is not how one specifies hook dependencies, see: https://react.dev/reference/react/useEffect#useeffect

Its comma-separated values that react reads out, not a boolean condition like here.

You can verify this easily by letting intellisense (the light bulb) in e.g. VSCode autofix missing deps here.